src/fidgets/ui/Video.tsx

```diff
@@ -1,18 +1,16 @@
-import React, { useEffect } from "react";
+import IFrameWidthSlider from "@/common/components/molecules/IframeScaleSlider";
 import TextInput from "@/common/components/molecules/TextInput";
 import {
   FidgetArgs,
-  FidgetProperties,
   FidgetModule,
+  FidgetProperties,
   type FidgetSettingsStyle,
 } from "@/common/fidgets";
-import { isValidUrl } from "@/common/lib/utils/url";
-import useSafeUrl from "@/common/lib/hooks/useSafeUrl";
-import { defaultStyleFields } from "@/fidgets/helpers";
-import IFrameWidthSlider from "@/common/components/molecules/IframeScaleSlider";
-import { transformUrl } from "@/fidgets/helpers";
```

Comment:
We need this transformUrl function to convert normal Youtube and Vimeo links into the embeddable format (ie. transforming https://www.youtube.com/watch?v=lOzCA7bZG_k to https://www.youtube.com/embed/lOzCA7bZG_k).

Please make sure users can still embed YouTube and Vimeo videos by pasting the normal URL